autoload/lsp/ui/vim.vim

```diff
@@ -239,25 +239,29 @@ function! s:handle_symbol(server, last_req_id, type, data) abort
     endif
 endfunction
 
-function! s:handle_location(server, last_req_id, type, data) abort
-    if a:last_req_id != s:last_req_id
-        return
-    endif
+function! s:handle_location(jump_if_one, ctx, server, type, data) abort
```

Comment:
can you document what ctx contains as comments. something like `" ctx = { counter, list  } `.